.github/workflows/tests.yml

```diff
@@ -8,4 +8,19 @@ jobs:
     - uses: actions/setup-ruby@v1
       with:
         ruby-version: '2.6'
-    - run: ruby tests/well-formed-yaml.rb
+    - uses: actions/cache@v1
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-gems-
+
+    - name: Bundle install
+      run: |
+        gem install bundler
```

Comment:
You shouldn't need this, Ruby 2.6.6 bundles bundler with it.

Reply:
```
/opt/hostedtoolcache/Ruby/2.6.6/x64/lib/ruby/2.6.0/rubygems.rb:283:in `find_spec_for_exe': Could not find 'bundler' (2.1.4) required by your /home/runner/work/govuk-coronavirus-content/govuk-coronavirus-content/Gemfile.lock. (Gem::GemNotFoundException)
To update to the latest version installed on your system, run `bundle update --bundler`.
To install the missing version, run `gem install bundler:2.1.4`
	from /opt/hostedtoolcache/Ruby/2.6.6/x64/lib/ruby/2.6.0/rubygems.rb:302:in `activate_bin_path'
	from /opt/hostedtoolcache/Ruby/2.6.6/x64/bin/bundle:23:in `<main>'
##[error]Process completed with exit code 1.
```

😭 y tho?

Reply:
So I removed it and the tests failed as above, so I've put it back.